tasks/size_contracts.js

```diff
@@ -82,7 +82,14 @@ task(
   }));
 
   if (config.alphaSort) {
-    outputData.sort((a, b) => a.displayName.toUpperCase() > b.displayName.toUpperCase() ? 1 : -1);
+    outputData.sort((a, b) => {
+      if (a.displayName.toUpperCase() > b.displayName.toUpperCase())
+        return 1
+      else if (a.displayName.toUpperCase() < b.displayName.toUpperCase())
+        return -1;
+
+      return a.deploySize == b.deploySize ? a.initSize - b.initSize : a.deploySize - b.deploySize;
```

Comment:
The case that this addresses is so specific that most users would never even notice it, and the exact behaviour would be difficult to document.